File: 2824. Count Pairs Whose Sum is Less than Target/2824. Count Pairs Whose Sum is Less than Target.py
class Solution:
    def countPairs(self, nums: List[int], target: int) -> int:
        n=len(nums)
        count=0
        for i in range(n-1):
            for j in range(1,n):
                if i<j and nums[i]+nums[j]<target:
                    count+=1
        return count

File: 2824. Count Pairs Whose Sum is Less than Target/2824. Count Pairs Whose Sum is Less than Target.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var countPairs = function(nums, target) {
    var n=nums.length
    var count=0
    for(let i=0;i<n;i++){
        for(let j=0;j<n;j++){
            if(i<j && nums[i]+nums[j]<target){
                count++
            }
        }
    }
    return count    
};